versioned_docs/version-V2/glossary.md

```diff
@@ -4,7 +4,7 @@ sidebar_position: 7
 
 # Glossary
 
-## Identity commitment
+## identity commitment
```

Comment:
Is this a typo?

Reply:
> Is this a typo?

No. For glossary terms, I only want uppercase for proper nouns so users know which are proper and which aren't.